addons/mail/models/mail_activity_mixin.py

```diff
@@ -395,6 +395,16 @@ def activity_schedule(self, act_type_xmlid='', date_deadline=None, summary='', n
 
         model_id = self.env['ir.model']._get(self._name).id
         create_vals_list = []
+        if not self._ids:
```

Comment:
Wait, what ? Is it because it is used to create activities on "no model" (linked to the other comment about using mixin by itself) ? Feels weird, mixin is not intended to be used like that. Would move that where it belongs.